com/williamfiset/algorithms/sorting/RadixSort.java

```diff
@@ -1,12 +1,73 @@
-/**
- * NOTE: This file is still under development
- **/
-package com.williamfiset.algorithms.sorting;
-
-public class RadixSort {
-
-  public static void radixSort(int[] ar) {
-    
-  }
-
-}
+

```

Comment:
Can you add a few tests in a new file called at javatests/com/williamfiset/algorithms/sorting/RadixSortTest.java?
Make sure the new file has the package name javatests.com.williamfiset.algorithms.sorting and run your tests using gradle like "gradle test --tests javatests.com.williamfiset.algorithms.sorting.RadixSortTest". There are more instructions in the [README](https://github.com/williamfiset/Algorithms#adding-a-new-algorithm).

Reply:
@williamfiset 
I am using JUnit4 testing, is it okay? I don't particularly know how to use gradle? Is that okay.
Please, guide me.